creators/extension/pkg/vue-shim.ts

```diff
@@ -1,4 +1,15 @@
-declare module '*.vue' {
-  import Vue from 'vue';
-  export default Vue;
+export {}
```

Comment:
This empty export is in line with guidance from the Vue docs[^1] on the topic of augmenting vue types.

[^1]: https://vuejs.org/guide/typescript/options-api#type-augmentation-placement